stepprogress/src/main/java/com/funrisestudio/stepprogress/StepProgressView.kt

```diff
@@ -505,6 +505,63 @@ class StepProgressView : ViewGroup, View.OnClickListener {
         return stepProgress.isAllDone()
     }
 
+    /**
+     * Change title text padding
+     */
+    fun setTextTitlePadding(size: Float) {
+        textTitlePadding = size
+        requestLayout()
+    }
+
+    /**
+     * Change node title text size
+     */
+    fun setNodeTitleSize(size: Int) {
+        textNodeTitleSize = size
+        resetView()
+        invalidate()
```

Comment:
resetView() updates children and invalidates view. So you don't need to call invalidate() again

Reply:
fixed